.travis.yml

```diff
@@ -9,7 +9,6 @@ matrix:
       sudo: true
     - python: 3.6
     - python: 3.5
-    - python: 2.7
```

Comment:
I would suggest removing Python 3.5 and 3.6 here as well, for the simple reason that we shouldn't be using Travis at all any more, and this conserves a bit of runtime until we can replace it with Azure